README.md

```diff
@@ -14,91 +14,132 @@ another Hadoop file system that executes it against the underlying object store.
 #### Pre-requisites 
 - Install maven 
 
-#### Build instructions
+#### Steps
 
-1. Clone the repo with 
+1. Clone the repo:
     ```shell
     git clone git@github.com:treeverse/hadoop-router-fs.git
     ```
 
-2. Build with maven
-```shell
-mvn clean install
-```
+2. Build with maven:
+   ```shell
+   mvn clean install
+   ```
 
 ## How to configure RouterFS 
 
 ### Configure Spark to use RouterFS
 
-You should tell Spark to use RouterFS as the file system for URIs you would like to transform at runtime. To do that, 
-add the following Spark property to your Spark configurations: 
+You should instruct Spark to use RouterFS as the file system implementation for the URIs you would like to transform at runtime. To do that, 
+add the following property to your Spark configurations: 
 ```properties
 fs.${fromFsScheme}.impl=RouterFileSystem
```

Comment:
Does this work?  Don't I need to set the qualified classname, with package and all?

Reply:
true